spring-cloud-task-integration-tests/src/test/java/org/springframework/cloud/task/listener/BatchExecutionEventTests.java

```diff
@@ -240,7 +241,7 @@ public void receive(Object exceptionMessage) {
 	public static class ItemWriteEventsListenerBinding {
 
 		@StreamListener(Sink.INPUT)
-		public void receive(Object itemWrite) {
+		public void receive(String itemWrite) {
 			assertTrue("Message should start with '3 items'", itemWrite.toString().startsWith("3 items "));
```

Comment:
@mbogoevici What I'm specifically asking is that, before `Object` worked.  Now we're being required to specify the type (the test fails without this change).